src/epic-cli.cpp

```diff
@@ -139,7 +139,6 @@ int main(int argc, char** argv) {
         auto client = CreateClient("0.0.0.0", rpc_port);
 
         switch (commandMap.at(command)) {
-
```

Comment:
this line is added by design

Reply:
Removed by clang-format. Add `// clang-format off` ?